uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-create-dropdown.html

```diff
@@ -0,0 +1,16 @@
+<div class="btn-group" uib-dropdown>
+  <button type="button" class="btn {{vm.btnClass}} dropdown-toggle" uib-dropdown-toggle>
+    <i class="fa fa-plus-circle" aria-hidden="true"></i> Record New Absence
+  </button>
+  <ul class="dropdown-menu" role="menu" uib-dropdown-menu>
+    <li ng-repeat="option in vm.leaveRequestOptions">
+      <a
+        href="#"
+        leave-request-popup leave-type="{{option.type}}"
```

Comment:
since every other attribute is in its own line, `leave-type="{{option.type}}"` should be on its own as well